Makefile

```diff
@@ -1,23 +1,29 @@
 DEV_ROCKS=busted luacov luacov-coveralls luacheck ldoc
 BUSTED_ARGS ?= -v -o gtest
 CASSANDRA ?= 3.10
+PROD_ROCKFILE = $(shell ls lua-cassandra-*.rockspec | grep -v dev)
+DEV_ROCKFILE = $(shell ls lua-cassandra-*.rockspec | grep dev)
+FLAGS ?=
 
 .PHONY: install dev busted prove test clean coverage lint doc
 
 install:
-	@luarocks make
+	@luarocks make $(PROD_ROCKFILE)
```

Comment:
Works for me without specifying the rockspec:

```
$ pwd
lua-cassandra
$ luarocks --version
/usr/local/bin/luarocks 2.4.2
LuaRocks main command-line interface
$ luarocks make
lua-cassandra 1.2.3-0 is now installed in /usr/local/opt/kong (license: MIT)
```

Reply:
Looks like this changed in luarocks 2.4.x. On my system I had luarocks 2.3 and that one fails.
`luarocks install luarocks` fixed it.

Will revert